workshops/workshop_june_2024/newspaper-metadata.ipynb

```diff
@@ -0,0 +1,1326 @@
+{
```

Comment:
<p>After this, I'd showing how to add the text as a column to the items_sample (also so that it looks more like the df Kaspar and I will use). Something like this:</p><p> </p><blockquote>Finally, you can add the extracted text as a column in the item_sample dataframe:</blockquote><p> </p>

<pre class="ql-syntax" spellcheck="false">items_sample['text'] = items_sample['item_code'].map(items_text)
items_sample
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Living-with-machines/newspapers/pull/3/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='110'/>